modules/ROOT/pages/dw-core-functions-reduce.adoc

```diff
@@ -8,7 +8,7 @@ endif::[]
 [[reduce1]]
 == reduce&#40;Array<T&#62;, &#40;element: T, accumulator: T&#41; &#45;&#62; T&#41;: T &#124; Null
 
-For each element of the input array, in order, applies the reduction lambda expression (function) then replaces the accumulator with the new result. The lambda expression can use both the current input array element and the current accumulator value. 
+For each element of the input array, in order, applies the reduction lambda expression (function) then replaces the accumulator with the new result. The lambda expression can use both the current input array element and the current accumulator value.
```

Comment:
This sentence doesn't read well. I think it would read better if it started with "Each" instead of "For." I would also say "and then" with a comma, but I see that Grammarly doesn't like the comma.

Each element of the input array, in order, applies the reduction lambda expression (function), and then replaces the accumulator with the new result. The lambda expression can use both the current input array element and the current accumulator value.

Reply:
reworded